### DISCUSSION TITLE
How to obtain updated translation files?

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#12293] [Q&A] [answered]
Good morning, 

I have done some translations in the Orchard Core crowdin's project but if I download the .po files seems a version before my modifications. 

How can I get the latest version? Have I to ask about "build" the translations in the crowdin's project? 

Thank you.

PS. I don't wanna "push you" to build the translations, I want to understand the iter.

**Comment by agriffard** [comment, marked as answer]
The translations on Crowdin need indeed to be built to appear in the zip file.
I just did it so you can try them.

We don't have a real strategy apart from updating them before a new OC release..

**Reply by ns8482e** [reply]
May be we should have some revision updates for current version on some interval (weekly/biweekly) so that newly updated transactions can be used with currently released version

**Reply by PiemP** [reply]
Could be really useful...but I believe could be useful have something to sync translation's files and the Orchard Core releases: if this "something" doesn't already exist.

Personally, in this moment, I have edited my publish profile to retrieve updated translations from a folder inside my project. Because Orchard Core retrieve the "release translations" when I try to build the project. I don't know if it is a good practice but is a solution for me 😁.

Is it possible to tag the translations to a specific Orchard Core's version? Because in this way we can match an Orchard Core release to a specific set of translation's files.
